api/http/management.go

```diff
@@ -154,7 +155,7 @@ func (h ManagementController) ConnectDevice(
 			rest.RenderError(c, s, e)
 		},
 	}
```

Comment:
Can you change the `Upgrader.Subprotocols` from `[]string{"binary"}` to `[]string{"protomsg/msgpack"}`?

Reply:
[sure](https://github.com/mendersoftware/deviceconnect/pull/13/commits/03dd6c7c927d58219cb608e8f1f708c78bd11362) man